index.html

```diff
@@ -536,34 +536,61 @@
-feature should not then be used for additional secondary [=purposes=]. Email is often used
-for login and communication [=purposes=], including essential transactional interactions.
-Using emails for cross-context recognition [=purposes=] is therefore not only
-[=inappropriate=] but also threatens key messaging infrastructure. Future iterations of
-the Web platform should ensure that users can log into sites and receive communication
-without relying on email at all.
+A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define them. Their identity
+*in a [=context=]* is the set of characteristics they present to that context. People
+frequently present different identities to different contexts, and also frequently share an identity
+between several contexts.
+
+<dfn>Cross-context recognition</dfn> is the act of recognising that an [=identity=] in one
+[=context=] is the same [=person=] as an [=identity=] in another [=context=]. [=Cross-context
+recognition=] can at times be [=appropriate=] but anyone who does it needs to be careful not to
+apply the [=norms=] of one [=context=] in ways that violate the [=norms=] around use of information
+acquired in a different [=context=]. (For example, if you meet your therapist at a cocktail party,
+you expect them to have rather different discussion topics with you than they usually would, and
+possibly even to pretend they do not know you.) This is particularly true for [=vulnerable=] people
+as recognising them in different [=contexts=] may force their vulnerability into the open.
+
+In computer systems and on the Web, an [=identity=] seen by a particular website is typically
+assigned an <dfn>identifier</dfn> of some type, which makes it easier for an automated system to
+store data about that user.
+
+<div class="practice">
+
+<p><span class="practicelab" id="principle-identity-per-context">[=User agents=] should support
+their [=users=]' [=autonomy=] by helping them present their intended [=identity=] to each
+[=context=] that they visit.</span></p>
+
+</div>
+
+To do this, [=user agents=] have to make some assumptions about the borders between [=contexts=]. By
+default, [=user agents=] define a <dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+
+* A set of [=environments=] (roughly iframes (including cross-site iframes), workers, and top-level
+  pages)
+* whose [=environment/top-level origins=] are in the [=same site=] (roughly, their [=origin/hosts=]
+  have the same [=host/registrable domain=], but see [[PSL-PROBLEMS]])
+* being visited within the same user agent installation (and profile for user agents that support
+  multiple profiles)
+* between points in time that the user or their agent clears that [=site=]'s cookies and other
+  storage (which is sometimes automatic at the end of each session).
+
+Even though this is the default, [=user agents=] are free to both widen and restrict this context as
```

Comment:
Should this text be part of the document? First party sets are still pretty well contested across other vendors; I'm not sure we should acknowledge them.

Reply:
I agree with this, and @jonathanKingston has put more directly and concisely what I tried to mention on the call:

If there are context boundaries that are controversial, or might be ruled out by the principals mentioned elsewhere in the document, I dont think we should include those context boundaries in a menu of examples.

Reply:
I agree that FPS are still heavily contested, however I don't want to just drop things that are contentious. We should be using documents like this one to hash things out.

This seems like a good place to flag an in-document issue, no?

```
☣️⚠️☢️ WARNING ☣️⚠️☢️
People disagree on boundaries yada yada…
```

Reply:
There are several examples of user-serving identity partitions that are excluded by the purely-site-based definition above here. I understand that Mozilla's currently shipping [one of them](https://github.com/disconnectme/disconnect-tracking-protection/blob/master/entities.json), even though it would like to get rid of that in the long run. Others include:

* bbc.com/bbc.co.uk
* github.com/githubusercontent.com and google.com/googleusercontent.com, where the domains are separate for security reasons, but users still want a uniform identity across the pair in order to access ACL'ed content
* news.bbc/sports.bbc where the registrable-domain algorithm doesn't understand that a whole TLD can be a very expensive domain registration

My text doesn't endorse any particular solution to the problem, and it restricts solutions to ones where "the user understands [the sites] represent a single party", but it does say that it's reasonable for browsers to explore solutions. 

I'm happy to mark this as contentious if y'all still think the document should forbid browsers from working on this problem as a whole.

This paragraph also gives the example of users sync'ing their identity<->site mapping between devices. That can't be done by naively sync'ing all cookie values, but is that kind of goal also contentious?

Reply:
The idea is not to forbid browsers from working on this (at all), but on the contrary to mark this as an area that needs significant further discussion. We do want to get to a resolution before the document is finalised, though.

Reply:
Not to get too weedsy on the discussion regarding FPS but Mozilla's implementation is [off by default](https://github.com/w3ctag/design-reviews/issues/342#issuecomment-820551099) purely a web compat solution and they [don't wish to see it standardised](https://github.com/privacycg/proposals/issues/17#issuecomment-641691610).

Either way I'm happy to just mark this as "in discussion" and we should resolve before publishing.

Reply:
I've moved the mention of widening partitions into a "This is controversial" issue block. How's it look now?

Reply:
Discussion at the [Aug 18 meeting](https://github.com/w3ctag/privacy-principles/blob/main/meetings/2021-08-18-minutes.md#identity-and-boundary) indicated this was good enough for now. I'll merge tomorrow to give a bit of time for more asynchronous comments.